mlir/lib/Dialect/Tosa/Transforms/TosaValidation.cpp

```diff
@@ -509,7 +509,8 @@ bool TosaValidation::isValidElementType(Type type) {
   if (isa<FloatType>(type)) {
     if (profile == TosaProfileEnum::BaseInference)
       return false;
-    return type.isF32() || type.isF16() || type.isBF16();
```

Comment:
The TOSA spec only supports the OCP versions of the FP8 formats, and this pass is verifying adherence to the spec, so we should only add the Float8E4M3FNType and Float8E5M2Type in this location.